extension/index.googleFunction.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * If you are deploying extension module as a serverless function,
+ * this will be the main javascript file.
```

Comment:
Maybe just put a link to the gcp docs and remove these confusing comments.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/476f9545083895617aaa53af788a20e932b90983